dashboard/src/pages/packages/index.vue

```diff
@@ -1,44 +1,116 @@
 <script setup lang="ts">
-import { PackageListStatusEnum } from "@/openapi-generator";
+import type { PackageListStatusEnum } from "@/openapi-generator";
+
 import PackageListLegend from "@/components/PackageListLegend.vue";
 import PageLoadingAlert from "@/components/PageLoadingAlert.vue";
 import StatusBadge from "@/components/StatusBadge.vue";
+import Tabs from "@/components/Tabs.vue";
+import Tooltip from "bootstrap/js/dist/tooltip";
 import UUID from "@/components/UUID.vue";
+import { onMounted } from "vue";
+import { useAsyncState } from "@vueuse/core";
 import { useAuthStore } from "@/stores/auth";
 import { useLayoutStore } from "@/stores/layout";
 import { usePackageStore } from "@/stores/package";
-import { useAsyncState } from "@vueuse/core";
-import Tooltip from "bootstrap/js/dist/tooltip";
-import { onMounted } from "vue";
+import { useRoute, useRouter } from "vue-router/auto";
+import { watch } from "vue";
+
+// General icons.
 import IconInfoFill from "~icons/akar-icons/info-fill";
 import IconBundleLine from "~icons/clarity/bundle-line";
+
+// Pager icons.
 import IconSkipEndFill from "~icons/bi/skip-end-fill";
 import IconSkipStartFill from "~icons/bi/skip-start-fill";
 import IconCaretRightFill from "~icons/bi/caret-right-fill";
 import IconCaretLeftFill from "~icons/bi/caret-left-fill";
 
+// Tab icons.
+import RawIconCheckCircleLine from "~icons/clarity/check-circle-line?raw&font-size=20px";
+import RawIconTimesCircleLine from "~icons/clarity/times-circle-line?raw&font-size=20px";
+import RawIconPlayLine from "~icons/clarity/play-line?raw&font-size=20px";
+import RawIconBarsLine from "~icons/clarity/bars-line?raw&font-size=20px";
```

Comment:
I have been thinking this for a while, we may want to look for some import sorting tool.

Reply:
Yeah, the imports are pretty messy.